server/app/controllers/meetings_controller.rb

```diff
@@ -63,7 +63,7 @@ def show
   # POST /meetings/
   def create
     # TODO: handle creation by company representatives
-    if !confirm_requester_is_admin
+    if !confirm_requester_is_admin && current_user.usertype != "representative"
```

Comment:
I think I found it:
Change the order of these two. `confirm_requester_is_admin` renders something if it fails. So move the company representative part forward

Reply:
And `&&`->`||`

Reply:
We haven't made the switch from "company representative" to "representative" yet I don't think

Reply:
Thank you Insoo and Ryan for comments. I think I have solved all the above. actually && works than ||

Reply:
Yep you are right